app/facade/foods_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodsFacade
+  def self.create_food(query)
+    json = FoodsService.get_food_data(query)
+    json[:foods].map do |data|
+      Food.new(data)
+    end.take(10)
+  end
+
+  def self.returned_items_number(query)
+    json = FoodsService.get_food_data(query)
+    json[:totalHits]
```

Comment:
Same deal as with parks finder. You can add this to the method above and use the same API call to extract the total. Then, just return the foods and the total as an array so the controller can receive both:
`[foods, total] = FoodsFacade.create_food(params[:query])`